build.gradle

```diff
@@ -138,6 +144,7 @@ if (useDownloadedLibs) {
         src {
             switch (DefaultPlatform.RESOURCE_PREFIX) {
                 case 'darwin':
+                case 'darwin-x86-64':
```

Comment:
The name of the platform has changed in the latest release of JNA, for reasons to do with the M1 arm64 silicon stuff.